packit/command_handler.py

```diff
@@ -124,8 +124,17 @@ class SandcastleCommandHandler(CommandHandler):
 
     name = RunCommandType.sandcastle
 
-    def __init__(self, config: Config, local_project: Optional[LocalProject] = None):
-        super().__init__(local_project=local_project, config=config)
```

Comment:
The issue is that we used to deduce the `working_dir` from the `local_project`, right?

Reply:
that's right :+1: